package.json

```
@@ -113,5 +113,6 @@
   },
   "dependencies": {
     "chance": "^1.1.8"
-  }
+  },
+  "packageManager": "yarn@1.22.22+sha1.ac34549e6aa8e7ead463a7407e1c7390f61a6610"
```

```suggestion
}
```

Can you remove this?